src/env.ts

```diff
@@ -0,0 +1,99 @@
+import { JSONSchema } from '@apidevtools/json-schema-ref-parser';
+import _ from 'lodash';
```

Comment:
maybe I am too conservative, but I dislike importing lodash as `_` , I prefer to name it lodash

Reply:
explicited